src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -4,62 +4,100 @@ layout: default
+3. Your `infusion` directory will include a single file containing all of the JavaScript you need (`infusion-uiOptions.js`), HTML templates, CSS files, and other components to get UI Options to work. You will later link to these files in your HTML files.
+4. Now that `infusion` is in your project directory, you can delete the `infusion-uiOptions-X.X.X.zip` (or similar name) from your download directory.
+
+###  Get Infusion UI Options Stable Release with `npm`
+This method will download and install the most recent stable release of Infusion.
+
+1. From your project directory, run `npm i infusion`.
+2. Within the `node_modules/infusion/` directory in your project, run `npm install` to get Infusion's dependencies.
+3. Then build a version of Infusion with UI Options by running `grunt custom --include="uiOptions" --name="uiOptions"`. This will create a build of Infusion in the `node_modules/infusion/build/` directory. Note: [Get Grunt](https://gruntjs.com/) if you don't have it installed.
+4. Your `node_modules/infusion/build` folder will include a single file containing all of the JavaScript you need
    (`infusion-uiOptions.js`), HTML templates, CSS files, and other components to get UI Options to work. You will later
    link to these files in your HTML files.
-4. Now that `infusion` is in your project directory, you can delete the `infusion-uiOptions-2.0.0.zip` (or similar name)
-   from your download directory.
+5. Rename the directory from `build` to `infusion`, and move (or copy) the `build` directory to your project (i.e. `my-project/lib/infusion`).
+6. Optional: You can now safely delete the `infusion` directory created in step 1.
+
+Summary of commands:
+```bash
+npm i infusion
+cd node_modules/infusion/
+grunt custom --include="uiOptions" --name="uiOptions"
+mv build/ my-project/lib/infusion
+```
+
+### Infusion Latest Development Source
+This method will download and install the current version of Infusion in development.
+
+<div class="infusion-docs-note">
+<strong>Note:</strong>
+Use the latest development source if you want to try out the latest changes and features. We recommend using a stable release for production situations by using one of the other methods above.
+</div>
+
+1. Clone the Infusion GitHub repository anywhere to your local filesystem: `git clone https://github.com/fluid-project/infusion`. This will create a directory `infusion` containing the Infusion source code.
+2. Within the `infusion` directory, run `npm install` to get Infusion's dependencies.
+3. Build a version of Infusion with UI Options by running `grunt custom --include="uiOptions" --name="uiOptions"`. This will create a build of Infusion in the directory `infusion/build/`. Note: [Get Grunt](https://gruntjs.com/) if you don't have it installed.
```

Comment:
Instead of having notes here, I think it would make sense to have a pre-requisite or dependencies section at the top of each of these sections. Then someone coming to them knows what's expected for them to have installed etc. before embarking on an installation type.

Reply:
I just checked `package.json` and grunt is a dev dependency... so if the Note even applicable? Grunt would have been installed with `npm install` correct?